### PR TITLE
fix: LinkedIn Logo

### LINKED ISSUE #6928: icon: need update LinkedIn
## Issue:

We are violating LinkedIn Trademark ! Here the LinkedIn [trademark policy](https://brand.linkedin.com/content/brand/global/en_us/index/visual-identity/logo)

## Solution:

Update the component [`components/Icons/Social/LinkedIn.tsx`](https://github.com/nodejs/nodejs.org/blob/main/apps/site/components/Icons/Social/LinkedIn.tsx)

Here svg extract from LinkedIn website:
_(yeah they didn't furnish svg)_

```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="mercado-match" width="24" height="24" focusable="false">
      <path d="M20.5 2h-17A1.5 1.5 0 002 3.5v17A1.5 1.5 0 003.5 22h17a1.5 1.5 0 001.5-1.5v-17A1.5 1.5 0 0020.5 2zM8 19H5v-9h3zM6.5 8.25A1.75 1.75 0 118.3 6.5a1.78 1.78 0 01-1.8 1.75zM19 19h-3v-4.74c0-1.42-.6-1.93-1.38-1.93A1.74 1.74 0 0013 14.19a.66.66 0 000 .14V19h-3v-9h2.9v1.3a3.11 3.11 0 012.7-1.4c1.55 0 3.36.86 3.36 3.66z"></path>
</svg>
```

### PR DESCRIPTION
**Pull request by Jay-Karia**
### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npm run format` to ensure the code follows the style guide.
- [x] I have run `npm run test` to check if all tests are passing.
- [x] I have run `npx turbo build` to check if the website builds without errors.
- [x] I've covered new added functionality with unit tests if necessary.

Closes #6928

### CODE REVIEW

File: apps/site/components/Icons/Social/LinkedIn.tsx [hunk 1]
```diff
@@ -1,32 +1,9 @@
...
-          fill="white"
-        />
-        <path
-          d="M10.9426 7.59155H8.73343V14.2858H10.9426C10.9426 14.2858 10.9426 12.1783 10.9426 10.8607C10.9426 10.0698 11.2126 9.27544 12.2901 9.27544C13.5079 9.27544 13.5005 10.3104 13.4949 11.1123C13.4874 12.1604 13.5052 13.2299 13.5052 14.2858H15.7143V10.7527C15.6956 8.49675 15.1077 7.45725 13.1738 7.45725C12.0253 7.45725 11.3134 7.97867 10.9426 8.4504V7.59155Z"
-          fill="white"
-        />
-      </g>
-    </g>
-  </svg>
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="mercado-match" width="24" height="24" focusable="false">
```
Comment (AugustinMauroy) [inline apps/site/components/Icons/Social/LinkedIn.tsx]:
```suggestion
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" width="24" height="24" {...props}>
```

File: apps/site/components/Icons/Social/LinkedIn.tsx [hunk 2]
```diff
@@ -1,32 +1,9 @@
...
-          fill="white"
-        />
-        <path
-          d="M10.9426 7.59155H8.73343V14.2858H10.9426C10.9426 14.2858 10.9426 12.1783 10.9426 10.8607C10.9426 10.0698 11.2126 9.27544 12.2901 9.27544C13.5079 9.27544 13.5005 10.3104 13.4949 11.1123C13.4874 12.1604 13.5052 13.2299 13.5052 14.2858H15.7143V10.7527C15.6956 8.49675 15.1077 7.45725 13.1738 7.45725C12.0253 7.45725 11.3134 7.97867 10.9426 8.4504V7.59155Z"
-          fill="white"
-        />
-      </g>
-    </g>
-  </svg>
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" width="24" height="24" {...props}>
```
Comment (canerakdas) [inline apps/site/components/Icons/Social/LinkedIn.tsx]: You need to use `npm run format` to ensure the code follows the style guide. AFAIK, our prettier settings don't allow this many HTML attributes in the same line

Comment (Jay-Karia) [reply apps/site/components/Icons/Social/LinkedIn.tsx]: Ok 👍

**Comment by canerakdas** [comment]
IMO, the error we got in [Build](https://github.com/nodejs/nodejs.org/actions/runs/10004099056/job/27652164362?pr=6932) should be solved with #6933. Could you rebase please @Jay-Karia 🙇

**Review by canerakdas — APPROVED** [review]
LGTM! Thank you for your contribution :shipit:

**Comment by AugustinMauroy** [comment]
I didn't understand all of LinkedIn's trademark policy, but shouldn't the logo always be in blue?
`#3966AD` (it's the blue of the logo)

**Comment by Jay-Karia** [comment]
Should we merge this ?? 🙃

**Comment by AugustinMauroy** [comment]
Why not but we need to answer to my questions

**Comment by ovflowd** [comment]
> erstand all of LinkedIn's trademark policy, but shouldn't the logo always be in blue?
> `#3966AD` (it's the blue of the logo)

Right the logo should be blue, not black.

**Comment by canerakdas** [comment]
Can't we use the monochrome version(Black, Reversed white) of the LinkedIn logo? When I look at the contrast ratio (`#3966AD`), it does not seem to provide a sufficient contrast ratio for the our dark theme;

image


I think it would be better to use monochrome versions as we can provide a better contrast ratio 👀  What do you think @ovflowd @AugustinMauroy

**Comment by ovflowd** [comment]
> Can't we use the monochrome version(Black, Reversed white) of the LinkedIn logo? When I look at the contrast ratio (`#3966AD`), it does not seem to provide a sufficient contrast ratio for the our dark theme;
> 
> image
> I think it would be better to use monochrome versions as we can provide a better contrast ratio 👀 What do you think @ovflowd @AugustinMauroy

Are those official/allowed LinkedIn contrast schemas? We can't change the logo... If yes, then let's proceed with it.

**Comment by Jay-Karia** [comment]
> We never approve the following types of requests because they violate our Terms of Service.  So please don’t ask permission to:
> •  Modify our trademarks or combine them with any other symbols, words, images, designs, or incorporate them into a slogan,

from the official LinkedIn branding policy
https://brand.linkedin.com/policies

**Comment by canerakdas** [comment]
> > Can't we use the monochrome version(Black, Reversed white) of the LinkedIn logo? When I look at the contrast ratio (`#3966AD`), it does not seem to provide a sufficient contrast ratio for the our dark theme;
> > image
> > I think it would be better to use monochrome versions as we can provide a better contrast ratio 👀 What do you think @ovflowd @AugustinMauroy
> 
> Are those official/allowed LinkedIn contrast schemas? We can't change the logo... If yes, then let's proceed with it.

When I look at the document, there are black and reversed white versions;

image

And the branding assets created for China also have black and reversed white versions. When I look at the available documents, I see only the white version. I think the safest way would be to go with the blue one and update it when we find a better version in the future 😞

**Comment by Jay-Karia** [comment]
Can we go for Trademark request ? (if possible)

**Comment by bmuenzenmeyer** [comment]
a black / white logo is common practice and explicitly called out - we are overthinking it if we think we must use only blue

> While our default logo is blue, use the black or white version on layouts that are black and white only. You can also use the white version on dark backgrounds so it's easy to see.

from https://brand.linkedin.com/downloads

**Review by bmuenzenmeyer — APPROVED** [review]
AS-IS - this have my approval

**Comment by bmuenzenmeyer** [comment]
merging this - it's adherent to my reading of the policy - no longer in a circle, and the color scheme is correct for light/dark use cases

**Pull request merged**